js/index.js

```diff
@@ -0,0 +1,35 @@
+
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
+import {
+  AppRegistry,
+  StatusBar
```

Comment:
Inconsistent whitespace. Someone in the group needs to install ESLint in the project.